### DISCUSSION TITLE
File not found. Yunohost

### DISCUSSION THREAD

**Discussion by josephflag** [YOURLS/YOURLS#3111] [General]
Hi, everything works.
It updates the database with the short links but when I go to try the link it returns a blank page with "File not found."
Mine is an installation in yunohost
Can You help me?

**Comment by dgw** [comment]
https://github.com/YOURLS/YOURLS/wiki/404-instead-of-redirection

**Reply by josephflag** [reply]
Thank you, I had seen it.
I've tried but to no avail.
Mine is an installation in Yunohost with a dedicated domain, I don't know if this information can be useful to solve the problem

**Reply by dgw** [reply]
Knowing that it's YunoHost helps, in that I already know it's given others issues. I suggest consulting the [YunoHost forum](https://forum.yunohost.org/) or IRC channel (#yunohost at Libera Chat). No one within the YOURLS project is involved with maintaining that installer.
